File: CardMasterSkin/Elements/SESquare.vb
Namespace Skins

    Public Class SESquare
        Inherits SERectangle

        Public Sub New(skin As Skin, size As Integer)
            Me.New(skin, 0, 0, size)
        End Sub

        Public Sub New(skin As Skin, x As Integer, y As Integer, size As Integer)
            MyBase.New(skin, x, y, size, size)
        End Sub

    End Class

End Namespace

File: CardMasterSkin/Elements/SESquare.cs
﻿using CardMasterSkin.Skins;

namespace CardMasterSkin.Elements
{
    public class SESquare : SERectangle
    {
        public SESquare(Skin skin, int size) : this(skin, 0, 0, size)
        { }

        public SESquare(Skin skin, int x, int y, int size) : base(skin, x, y, size, size)
        { }

    }
}